packages/lang/Python3/package.mk

```
@@ -130,6 +130,8 @@ post_makeinstall_target() {
   rm -rf $INSTALL/usr/bin/python*-config
   rm -rf $INSTALL/usr/bin/smtpd.py $INSTALL/usr/bin/smtpd.py.*
 
+  find $INSTALL -name \*.o -delete
```

```suggestion
  find $INSTALL -name '*.o' -delete
```
should work too, but without the ugly escape.

---

changed as suggested